app/editor/src/features/admin/filters/FilterFormQuery.tsx

```diff
@@ -71,7 +71,7 @@ export const FilterFormQuery: React.FC = () => {
 
   return (
     <>
-      <Col>
+      <Col gap="0.5rem">
```

Comment:
couple of inline styles here, should they be moved to style classes?

Reply:
Not for these type of minor styling changes.  It's actually more work to setup random classes just for a basic gap.  Now, if we had some standardized css layout that everything inherited then I would entirely agree.  However, we've never received this, nor has it been implemented.